vendor/github.com/moby/term/.gitignore

```diff
@@ -2,7 +2,7 @@
 # global .gitignore or .git/info/exclude see https://help.github.com/articles/ignoring-files
 .*
 !.github
-!.gitignore
+#!.gitignore
```

Comment:
How did this get here?

Reply:
I was using a different clone and it was dirty :-/

Fixed....